xml/operating_system_darwin.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Patterns for common names of Mac OS X operating systems.
+-->
+<fingerprints matches="operating_system.name">
+  <!-- Match Mac OS Classic first due to weak matching on Mac OS X -->
+  <fingerprint pattern="^(?:Apple )?Mac OS ([7-9](?:\.\d+?)*?)$" flags="REG_ICASE">
```

Comment:
btw, given that this is a new fingerprint file, it would be great to see it not make #75 (see: #83 too) worse.  IOW, remove `flags` and just make the regular expression case insensitive (non-capture group the entire thing, toss in the `i` option, like `^(?i:(?:Apple )?Mac OS ([7-9](?:\.\d+?)*?))$`